src/ui/public/modals/modal_overlay.html

```diff
@@ -1 +1 @@
-<div class="kuiModalOverlay"></div>
+<div class="kuiModalOverlay" ng-keydown="onKeyDown($event)"></div>
```

Comment:
Since you're listening for the keypress event on `document.body`, you don't need this keydown directive here.